### ISSUE TITLE
Need assistance with vGPU not displaying after addons applied

### ISSUE THREAD

**Issue by psarinana-fsb** [harvester/harvester#6179]
I recently deployed Harvester v1.3.1 to our Supermicro X10DRT-PT with dual Xeon E5-2687W v4 @ 3.00GHz and 924G of ram.  I have a Tesla M60 installed that I am trying to setup for vGPU access:

`  *-display UNCLAIMED
       description: VGA compatible controller
       product: GM204GL [Tesla M60]
       vendor: NVIDIA Corporation
       physical id: 0
       bus info: pci@0000:84:00.0
       version: a1
       width: 64 bits
       clock: 33MHz
       capabilities: pm msi pciexpress vga_controller cap_list
       configuration: latency=0
       resources: memory:fa000000-faffffff memory:e0000000-efffffff memory:f0000000-f1ffffff ioport:f000(size=128)`
 
` *-display UNCLAIMED
       description: VGA compatible controller
       product: GM204GL [Tesla M60]
       vendor: NVIDIA Corporation
       physical id: 0
       bus info: pci@0000:85:00.0
       version: a1
       width: 64 bits
       clock: 33MHz
       capabilities: pm msi pciexpress vga_controller cap_list
       configuration: latency=0
       resources: memory:f9000000-f9ffffff memory:c0000000-cfffffff memory:d0000000-d1ffffff ioport:e000(size=128)`

I have verified that SR-IOV is enabled via bios settings and I have deployed the `[nvidia-driver-toolkit]` and `[pcidevices-controller]` addons.   Here is the configuration I used for the `[nvidia-driver-toolkit]` addon.

![image](https://github.com/user-attachments/assets/74d6ab74-02a0-4867-9546-25b19b8e0819)

Both the `[nvidia-driver-toolkit]` and `[pcidevices-controller]` addons show enabled.

![image](https://github.com/user-attachments/assets/9e359857-1771-40ed-9a75-f462f0c5b961)

But there are no SR-IOV GPU Devices shown 

![image](https://github.com/user-attachments/assets/d5a29492-454a-4caa-95a2-15e65abc0171)

![image](https://github.com/user-attachments/assets/419328db-4f3a-4617-a9a6-100e17a80320)

I can see SR-IOV Network devices so I believe the service is activated properly

![image](https://github.com/user-attachments/assets/4ad4ced4-b187-4217-afad-c77b888417ce)

Is there a specific step that I am missing to enable vGPU?  I have a support bundle that I can provide for my configuration.

Any assistance with this issue is very much appreciated.

**Comment by FrankYang0529** [comment]
Hi @psarinana-fsb, may you generate support-bundle for this issue?

https://docs.harvesterhci.io/v1.3/troubleshooting/harvester/#generate-a-support-bundle

cc @ibrokethecloud, could you take a look for this issue? Thank you.

**Comment by psarinana-fsb** [comment]
Certainly, attached is the support bundle for my configuration
[supportbundle_19b2afc0-d7ae-4be5-becc-c8b3d0ed9a04_2024-07-16T16-18-54Z.zip](https://github.com/user-attachments/files/16267474/supportbundle_19b2afc0-d7ae-4be5-becc-c8b3d0ed9a04_2024-07-16T16-18-54Z.zip)

**Comment by ibrokethecloud** [comment]
this is likely because the Tesla M60 is based on the older Maxwell architecture. NVIDIA sriov vgpu capability is available on GPU's based on Ampere architecture or later

**Comment by psarinana-fsb** [comment]
Thanks for the response.  I replaced the M60 with a T4 and I reinstalled Harvester.  I am still not seeing the vGPU resource.  Here is a screenshot of the PCI bios settings verifying SR-IOV:

![IMG_2173](https://github.com/user-attachments/assets/3e253075-b82c-4597-98e5-11ebc4b52c3e)

I see the TESLA T4 in PCI devices in Harvester

![image](https://github.com/user-attachments/assets/6d22f892-e048-4b46-b468-c0e1f3c551c8)

And I have verified the addons are enabled

![image](https://github.com/user-attachments/assets/cc8156a0-a6fb-4afa-ae73-6952660ef37f)

I am really not sure what I am missing here, I am providing another support bundle for this configuration.  Thanks again for looking into this

[supportbundle_efc4c818-c19b-4c04-81d6-b9ab660e9719_2024-07-31T15-42-00Z.zip](https://github.com/user-attachments/files/16444220/supportbundle_efc4c818-c19b-4c04-81d6-b9ab660e9719_2024-07-31T15-42-00Z.zip)
